gradle.properties

```diff
@@ -19,4 +19,23 @@ android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 
-detektVersion=1.18.0-RC2
\ No newline at end of file
+detektVersion=1.18.0-RC2
+
+
+POM_NAME=PayPal Payments - Android
+POM_PACKAGING=aar
+POM_DESCRIPTION=Paypal payment methods for Android
+POM_URL=not_defined
+
+POM_SCM_URL=https://github.com/paypal/Android-SDK
+POM_SCM_CONNECTION=scm:git:git://paypal/Android-SDK.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://paypal/Android-SDK.git
```

Comment:
Should these be using the `git@github` urls instead?

Reply:
i copied this from our checkout sdk, but then after reading a bit, it turns out that that is the way maven requires it: https://maven.apache.org/scm/git.html

Reply:
Does Sonatype require this too? In BT Android we use git@github. Not sure which is better here

Reply:
Me neither, i dont mind putting git@github